_people/lyu.md

```diff
@@ -0,0 +1,61 @@
+---
+# !!!!
+# The NAME of the file should be your LASTNAME (e.g. houben.md)
+# Do NOT delete any field, only fill in the required information
+# !!!! 
+
+# Your full name (Firstname, Lastname)
+name: Lijun, Lyu
```

Comment:
The comma (",") is not necessary. I would also update the template and remove it from the "example"